src/components/saas-list/saas-list.component.ts

```diff
@@ -0,0 +1,10 @@
+import {SaasListController} from './saas-list.controller';
+
+export const saasList: angular.IComponentOptions = {
+  bindings: {
+    title: '<?',
```

Comment:
If there are no SaaS Offerings defined, then the whole component should be hidden -title and all.  If title were separate, then additional code would be needed to hide the title as well as the component.